lib/MusicBrainz/Server/Entity/EditNote.pm

```diff
@@ -125,6 +125,7 @@ around TO_JSON => sub {
     my ($orig, $self) = @_;
 
     my $json = $self->$orig;
+    $json->{edit} = to_json_object($self->edit);
```

Comment:
We could choose to make edits part of linkedEntities if we worry that this is too much.